client/src/js/components/bhPatientFinancialActivity.js

```diff
@@ -67,6 +68,7 @@ function PatientFinancialActivityCtrl(Patients, moment, Session, Constants, $q)
         stockData.forEach((item, index) => {
           if (index < limitRecent) {
             dataMovement.push(item);
+            $ctrl.dataMovementTotalValue += item.value;
```

Comment:
This should sum all movements.  We need to know the total value, not just the value of 5 displayed.